core/base/src/TApplication.cxx

```diff
@@ -468,7 +468,46 @@ void TApplication::GetOptions(Int_t *argc, char **argv)
          } else {
             Warning("GetOptions", "-e must be followed by an expression.");
          }
+      } else if (!strcmp(argv[i], "--")) {
+         TObjString* macro = NULL;
 
+         if (fFiles) {
+            for (int f = 0; f < fFiles->GetEntriesFast(); f++) {
+               TObjString* file = dynamic_cast<TObjString*>(fFiles->At(f));
+
+               if (file->TestBit(kExpression))
+                  continue;
+               if (file->String().EndsWith(".root"))
+                  continue;
+               if (strchr(file->String().Data(), '('))
```

Comment:
Don't we have a TString function which can replace the `strchr`? Like `Index()` or something?